### ISSUE TITLE
Authority

### ISSUE THREAD

**Issue by AshkanA** [RinkeHoekstra/qber#68]
Authority – when assigning standardisation you don’t want to reuse a specific term from a _very_ specific dataset. For example searching for the standardized variable for religions we'll see a 1881 Scottish dataset. 

Ideal case, when creating a _new_ bottom up variable, it should be good to give it a CLARIAH uri which is more generic… so, it would be nice to create this: 

http://data.socialhistory.org/resource/Clariah/variable/RELIGION

instead of: 

http://data.socialhistory.org/resource/scotlcotland1881_5e-04/variable/RELIGION 

Doing so, in the future users could then select Clariah religion instead of building on a very specific standardisation which does not look very good when others are searching for religion.

**Comment by RinkeHoekstra** [comment]
I agree with the sketched problem, but the solution is not advisable as it will create clashes between URIs. The distinction between the database-specific and ‘generic’ variable is valuable (a dataset may have a very specific interpretation of a label for a variable, that does not hold for the entire repo). 

The perceived problem is that obscure terms are now presented alongside standard terms. The solution is to work out a ranking schema that pushes obscure URIs to the long tail (e.g. alongside the solution for issue #31), also it will help to not always produce a dataset specific URI as is currently done, but allow for _direct_ use of external URIs in the RDF serialisation.
